scripts/exglobal_atmos_sfcanl.sh

```diff
@@ -194,7 +194,8 @@ fi
 
 # Update surface restarts at middle of window
 for n in $(seq 1 ${ntiles}); do
-    ${NCP} ${COMIN_GES}/RESTART/${PDY}.${cyc}0000.sfc_data.tile${n}.nc ${COMOUT}/RESTART/${PDY}.${cyc}0000.sfcanl_data.tile${n}.nc
+#DONG    ${NCP} ${COMIN_GES}/RESTART/${PDY}.${cyc}0000.sfc_data.tile${n}.nc ${COMOUT}/RESTART/${PDY}.${cyc}0000.sfcanl_data.tile${n}.nc
```

Comment:
Why do we retain this commented out line?

Reply:
@RussTreadon-NOAA This is just for my notes in this draft PR, and I will remove it. Thanks.

Reply:
Good to know.  Thanks @jiaruidong2017